charts/amoro/values.yaml

```diff
@@ -0,0 +1,163 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# AMS server numbers
+replicaCount: 1
+
+image:
+  repository: arctic163/ams
+  pullPolicy: IfNotPresent
+  tag: ""
+
+imagePullSecrets: []
+#nameOverride: ""
+#fullnameOverride: ""
+
+# Role-based access control
+rbac:
+  # Specifies whether RBAC resources should be created
+  create: true
+  # RBAC rules
+  rules:
+    - apiGroups: [""]
+      resources: ["pods"]
+      verbs: ["create", "list", "delete"]
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # Specifies ServiceAccount name to be used (created if `create: true`)
+  name: ~
+
+server:
+  # AMS REST Service
+  rest:
+    enabled: true
+    port: 1630
+    service:
+      type: ClusterIP
+      port: "{{ .Values.server.rest.port }}"
+      nodePort: ~
+      annotations: {}
+  # AMS Thrift Server For table services
+  table:
+    enabled: true
+    port: 1260
+    service:
+      type: ClusterIP
+      port: "{{ .Values.server.table.port }}"
+      nodePort: ~
+      annotations: { }
+  # AMS Thrift Server For optimizing services
+  optimizing:
+    enabled: true
+    port: 1261
+    service:
+      type: ClusterIP
+      port: "{{ .Values.server.optimizing.port }}"
+      nodePort: ~
+      annotations: { }
+
+# $AMS_CONF_DIR directory
+amoroConfDir: /usr/local/ams/amoro-0.5.0/conf
```

Comment:
Same thing here as I comment in 'Chart.yaml', set a value to present the Amoro version and use the value instead of the number, like:
'amoroConfDir: /usr/local/ams/amoro-{{ .Values.appVersion }}/conf'.
But I'm not sure if this works fine. Appreciate it if you could verify it. This will save some upgrade work and avoid oversights.